### ISSUE TITLE
elevated apolipoprotein C-III level 

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/human-phenotype-ontology#6093]
**Preferred term label:**

elevated apolipoprotein C-III level 

**Synonyms**

elevated APOC3 level 

**Definition (free text, please give PubMed ID)**

An increased concentration in blood of apolipoprotein C-III, a protein component of triglyceride (TG)-rich lipoproteins (TRLs) including very low density lipoproteins (VLDL), high density lipoproteins (HDL) and chylomicrons. PMID:12006394

**Parent term (use hpo.jax.org/app)**

Abnormal apolipoprotein level (HP:0025201)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

0000-0002-9553-7227
--

**Comment by pnrobinson** [comment]
added term as HP:0033112, thank you!
